docs/technical-report-1.md

```diff
@@ -247,317 +247,317 @@ Limiting nodes to a maximum of one vote would likely be safe if the mean number
+#### MUSEN certificate  
+  
+[MUSEN](https://iohk.io/en/research/library/papers/musen-aggregatable-key-evolving-verifiable-random-functions-and-applications/) ("MUlti-Stage key-Evolving verifiable random fuNctions.") builds upon the concept of VRFs by introducing a key-evolving feature and allowing for aggregating VRF outputs. The aggregation capability promises to result in small certificates attesting to the Leios votes, MUSEN suffers from the need to record all of the verification keys (e.g., in a Merkle tree). MUSEN signature are approximately the same size as KES signatures. However, verification times may be too long for Leios. Overall, the benefit of MUSEN's key evolution is minor and its other resource costs may be greater than plain BLS.  
+  
+  
+### Insights regarding voting and certificates  
+  
+1. It is critically important to keep the size of votes small.  
+    1. Large votes can mean large certificates, and votes should also fit in one TCP MTU.  
+    2. Including KES signatures in votes means that votes will be 500-600 B in size.  
+    3. Optimizations might take advantage of the fact that KES periods, currently spanning 1.5 days on the Cardano mainnet, persist for many Leios voting cycles.  
+    4. Using ephemeral keys would make votes smaller, but then a key registration layer would have to be added to the protocol.  
+2. Neither ALBA, BLS, MUSEN, or their ZK variants are clear winners for the best choice of certificate scheme for Leios.  
+    1. Each has at least one strength and one drawback related to certificate size, proof time, or verification time.  
+    2. ALBA is close to being viable if vote size can reduced or if quorum disruption by adversaries with 10% of stake is acceptable.  
+    3. An ALBA security setting of $\ell_\text{sec} = \ell_\text{rel} = 80$ (i.e., eighty-bit security) seems sufficient for Leios voting.  
+    4. BLS is viable if ephemeral keys are used, but those would require pre-registration.  
+    5. BLS with KES keys could work if oversized Praos blocks (~160 kB) are allowed.  
+    6. ZK variants would result in small certificates but long proving times.  
+3. At least 500 votes and a 60% quorum will be needed.  
+    1. These parameters ensure voting security at least as strong as Praos security over a range of adversary strengths.  
+    3. ALBA would require a larger quorum.  
+    4. The clumpiness of the Cardano stake distribution on mainnet means that some producer nodes might cast more than one vote in a given pipeline.  
+    5. MUSEN and BLS certificates need further evaluation for Leios.  
+4. Generic benchmarks for cryptographic operations have provided guidance on the pros and cons of the prospective voting and certificate schemes, but further work on estimating CPU resources needed will require detailed implementation of the prospective voting and certificate schemes. For the time being, the following values can be used in simulation studies.  
+    1. Number of votes: 600  
```

Comment:
Isn't this 500?

Reply:
Yes, fixed in 1348e78690f6414bffd9f17ef3c51e611851d371.